Jenkinsfile

```diff
@@ -53,80 +55,25 @@ pipeline {
-                sh 'cp gitlab/dev/data/201706/sbr-2500-leu-paye-links.csv conf/sample/201706/sbr-2500-leu-paye-links.csv'
-                sh 'cp gitlab/dev/data/201706/sbr-2500-leu-vat-links.csv conf/sample/201706/sbr-2500-leu-vat-links.csv'
-
-                // Copy over the real data for August
-                sh 'cp gitlab/dev/data/201708/sbr-2500-ent-data.csv conf/sample/201708/sbr-2500-ent-data.csv'
-                sh 'cp gitlab/dev/data/201708/sbr-2500-ent-ch-links.csv conf/sample/201708/sbr-2500-ent-ch-links.csv'
-                sh 'cp gitlab/dev/data/201708/sbr-2500-ent-paye-links.csv conf/sample/201708/sbr-2500-ent-paye-links.csv'
-                sh 'cp gitlab/dev/data/201708/sbr-2500-ent-vat-links.csv conf/sample/201708/sbr-2500-ent-vat-links.csv'
-                sh 'cp gitlab/dev/data/201708/sbr-2500-ent-leu-links.csv conf/sample/201708/sbr-2500-ent-leu-links.csv'
-                sh 'cp gitlab/dev/data/201708/sbr-2500-leu-ch-links.csv conf/sample/201708/sbr-2500-leu-ch-links.csv'
-                sh 'cp gitlab/dev/data/201708/sbr-2500-leu-paye-links.csv conf/sample/201708/sbr-2500-leu-paye-links.csv'
-                sh 'cp gitlab/dev/data/201708/sbr-2500-leu-vat-links.csv conf/sample/201708/sbr-2500-leu-vat-links.csv'
                         
                 sh '$SBT clean compile "project api" universal:packageBin coverage test coverageReport'
                 stash name: 'compiled'
-                  
-                 // Remove Fake SQL data 
-                sh 'rm -rf conf/sample/ch_2500_data.sql'
-                sh 'rm -rf conf/sample/ent_2500_data.sql'
-                sh 'rm -rf conf/sample/leu_2500_data.sql'
-                sh 'rm -rf conf/sample/paye_2500_data.sql'
-                sh 'rm -rf conf/sample/unit_links_2500_data.sql'
-                sh 'rm -rf conf/sample/vat_2500_data.sql'
-
-                // Copy over real SQL data
-                sh 'cp gitlab/dev/data/sbr_inserts/ch_2500_data.sql conf/sample/ch_2500_data.sql'
-                sh 'cp gitlab/dev/data/sbr_inserts/ent_2500_data.sql conf/sample/ent_2500_data.sql'
-                sh 'cp gitlab/dev/data/sbr_inserts/leu_2500_data.sql conf/sample/leu_2500_data.sql'
-                sh 'cp gitlab/dev/data/sbr_inserts/paye_2500_data.sql conf/sample/paye_2500_data.sql'
-                sh 'cp gitlab/dev/data/sbr_inserts/unit_links_2500_data.sql conf/sample/unit_links_2500_data.sql'
-                sh 'cp gitlab/dev/data/sbr_inserts/vat_2500_data.sql conf/sample/vat_2500_data.sql'
               
                 script {
                     env.NODE_STAGE = "Build"
                     if (BRANCH_NAME == BRANCH_DEV) {
                         env.DEPLOY_NAME = DEPLOY_DEV
-                        sh "cp target/universal/${ORGANIZATION}-${MODULE_NAME}-*.zip ${DEPLOY_DEV}-${ORGANIZATION}-${MODULE_NAME}.zip"
+                        sh "cp target/universal/${MODULE_NAME}-*.zip ${DEPLOY_DEV}-${ORGANIZATION}-${MODULE_NAME}.zip"
```

Comment:
Revert back to '${ORGANIZATION}-...*' to keep conformity to other SBR apis, amend build file instead